extensions/wikia/GlobalNavigation/styles/GlobalNavigationAccountNavigation.scss

```diff
@@ -73,14 +76,24 @@
 		}
 	}
 
-	.sign-in-label {
+	.auth-label {
 		color: $global-navigation-text-color;
 		display: table-cell;
 		font-size: 12px;
 		padding-top: 2px;
 		vertical-align: middle;
 
-		&:hover {
+		&.register {
+			padding-left: 7px;
```

Comment:
Why don't we move padding-left to a higher declaration so we avoid repeating it for .sign-in? :)